nvflare/app_common/workflows/scatter_and_gather.py

```diff
@@ -171,6 +171,14 @@ def start_controller(self, fl_ctx: FLContext) -> None:
         fl_ctx.set_prop(AppConstants.START_ROUND, self._start_round, private=True, sticky=True)
         fl_ctx.set_prop(AppConstants.NUM_ROUNDS, self._num_rounds, private=True, sticky=False)
         self._global_weights = self.persistor.load(fl_ctx)
+        if self._global_weights.is_empty():
```

Comment:
According to the spec, persistor.load is returning a `Learnable`.

Learnable does not have "is_empty()" method